### ISSUE TITLE
question: how can you use future values of weather?

### ISSUE THREAD

**Issue by ihadanny** [nicolasfauchereau/Auckland_Cycling#1]
hey, thanks for this excellent notebook!
question1: how can you use future values of weather when forecasting? isn't that cheating? or is it an an average forecast according to the day in the year?
question2: why are you sufficing for a single test set and not running some sort of cross-validation, on several train/test splits? can't it be that you're drawing conclusions on an overfit?

**Comment by nicolasfauchereau** [comment]
Hi @ihadanny, sorry for the rather late reply. in reply to your questions: 
1: I here use observed values for the weather variables, but in the context of e.g. a full predictive model which would aim at predicting the cyclist numbers a few days in advance, you simply would use the outputs of a numerical weather prediction model 
2: because of the nature of the model, you cannot really do cross validation, as you need to respect the chronological ordering of the time-series. Not sure about your comment related to an overfit, the model does not 'see' the test data at any point, and the train vs test validation metrics are quite similar

**Comment by ihadanny** [comment]
1. got it
2. https://robjhyndman.com/hyndsight/tscv/
